### DISCUSSION TITLE
Add progress bar for SPI

### DISCUSSION THREAD

**Discussion by andimik** [KejPi/AbracaDABra#180] [Ideas]
On the console we have an output like

```
19:07:49.148 [I] SPIApp: 16: MOT directory NOT complete (decoded 60 / 65)
19:07:55.542 [I] SPIApp: 16: MOT directory NOT complete (decoded 61 / 65)
19:07:55.745 [I] SPIApp: 16: MOT directory NOT complete (decoded 62 / 65)
19:07:56.147 [I] SPIApp: 16: MOT directory NOT complete (decoded 63 / 65)
19:07:57.751 [I] SPIApp: 16: MOT directory NOT complete (decoded 64 / 65)
19:08:02.135 [I] SPIApp: 16: MOT directory complete
```

I'd like to propose a progress bar in the SPI window.

**Comment by KejPi** [comment]
What is SPI window?

**Reply by BoD-CZ** [reply]
Raději budu psát česky :-). 
Jde o o okno "Log aplikace".  U mě se to projevuje za dvou podmínek. Obě musí být splněny zároveň.
1) V nastavení programu není zatržen checkbox "Obrázky slideshow (SLS)", takže by se obrázky ukládat neměly. Pokud zatržen je, obrázky se korektně ukládají i když ta cesta je ve Win trochu nekorektní, protože nezačíná označením disku (např. C:) a lomítka nejsou zpětná (\) jak by ve Win měla být.
2) Děje se mi to pouze při příjmu polského MUXu.. Dá se to ověřit i na tom RAW souboru, který jsem posílal kvůli haváriím přijímaču na stanici Chopin.
Pokud se budete na něco chtít zeptat, klidně mi zavolejte. Můj telefon je v mailech ode mě. Hezký večer.

**Reply by KejPi** [reply]
Myslim, to s tematem nesouvisi, napisu na mail ;-)

**Reply by andimik** [reply]
> What is SPI window?

Sorry, I meant the EPG window.

**Comment by KejPi** [comment]
I do not think that EPG windows would be a good place for it because, it is only available when EPG on any service is available. This is not a common case, for instance in Austria I am not sure if there is EPG available from any ensemble. Typically only logos are broadcasted.  
But I think the idea of adding some progress is good. I just need to find some place for that. I have some idea, I will investigate if it is feasible.

**Comment by KejPi** [comment]
Done in 3.0.0.
